tla/consensus/Traceccfraft.tla

```diff
@@ -378,22 +368,6 @@ TraceNext ==
 
     \/ IsRcvProposeVoteRequest
 
-RaftDriverQuirks ==
-    \* The "nodes" command in raft scenarios causes N consecutive "add_configuration" log lines to be emitted,
-     \* where N is determined by the "nodes" parameter. At this stage, the nodes are in the "None" state.
-     \* However, the enablement condition of "ccfraft!Timeout" is only true for nodes in the "Candidate" or 
-     \* "Follower" state. Therefore, we include this action to address this quirk in the raft_driver.
-    \/ /\ IsEvent("add_configuration")
-       /\ state[logline.msg.state.node_id] = None
-       /\ configurations' = [ configurations EXCEPT ![logline.msg.state.node_id] = ToConfigurations(<<logline.msg.new_configuration>>)]
-       /\ state' = [ state EXCEPT ![logline.msg.state.node_id] = Follower ]
-       /\ UNCHANGED <<reconfigurationCount, removedFromConfiguration, messageVars, currentTerm, votedFor, candidateVars, leaderVars, logVars>>    
-    \/ /\ IsEvent("become_follower")
-       /\ state[logline.msg.state.node_id] = None
-       /\ configurations[logline.msg.state.node_id] = ToConfigurations(logline.msg.configurations)
-       /\ state' = [ state EXCEPT ![logline.msg.state.node_id] = Follower ]
-       /\ UNCHANGED <<reconfigurationVars, removedFromConfiguration, messageVars, currentTerm, votedFor, candidateVars, leaderVars, logVars>>
-
```

Comment:
\o/

Reply:
Well, now we have the more inoffensively named "bootstrap"... But it's at least a state snapshot after a realistic startup sequence that matches end to end, no more ex-nihilo configs. So I claim it's still progress.